packages/nextjs/app/page.tsx

```diff
@@ -10,7 +10,15 @@ import { useAccount } from "wagmi";
 
 const Home: NextPage = () => {
   const { address: connectedAddress } = useAccount();
-
+  const [isCheckInActive, setCheckInActive] = useState(false);
```

Comment:
This is used only to display buttons right?
there are two patterns here, one from react, with useState, which is correct, but mapbox is also using it's api to display markers and markers with attachment (https://docs.mapbox.com/help/tutorials/custom-markers-gl-js/#attach-popups-to-markers) I think we should go with the mapbox api to show items on the map, we can havea look together. Up to you!

Reply:
Tldr, The end of that issue should be a next link component directing to /checkin-form, or however you wanna call thos page

Reply:
This is used only to display buttons, yes. You'll see `setCheckInActive` removes the `disabled` state on the two button components. 

My instinct is to not use Mapbox for much interactivity — we should push that functionality out into off-map React components that we will have a lot more control over. i.e. for this MVP, the only thing you can do with a map is scroll / zoom around and tap to add a pin.